pages/artykuly/_categoryCode/_titleCode/_shortId.vue

```diff
@@ -53,6 +53,20 @@ export default {
       code: _.values(params).join('/'),
     })
   },
+  head () {
+    const headLength = 230
+    let articleHead = this.article.content.slice(0, headLength)
+    articleHead = articleHead.slice(0, articleHead.lastIndexOf(' '))
```

Comment:
You could export this two lines into function with meaningful name, and content, length as function parameters

Reply:
You can use elipsis helper. Or create your own fn named sentencify which slices the text to sentence with dot. Also please move the logic of creating metaDescription to prepareArticles in store.